content/news.md

```diff
@@ -1,21 +1,87 @@
 ---
 title: PyBaMM Newsroom
-shortcutDepth: 2
+shortcutDepth: 1
 newsHeader: "Battery Modelling Webinar Series 2021"
 date: 2021-07-06
+toc: true
 ---
 
 <!-- Format for adding news is: -->
 <!-- ## Title -->
-<!-- ###### Date -->
-<!-- Text -->
+<!-- _Date_ -->
+<!-- Content -->
 
 <!-- Change date and heading in YAML frontmatter for homepage -->
 
 <!-- Note: follow reverse order while updating, sort in descending order of recency -->
 
+## Release notes 23.9
+_October 31, 2023_
+
+PyBaMM 23.9 is finally here! This release marks the change to a new release schedule with three releases a year, aiming to publish the new versions at the end of January, May and September (ish). We would like to thank all the contributors that made this release possible.
+
+The full list of changes can be found in the CHANGELOG file, but here we provide a deeper overview of the main features of this release.
```

Comment:
Should we link CHANGELOG here?
```suggestion
The full list of changes can be found in the [CHANGELOG](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) file, but here we provide a deeper overview of the main features of this release.
```

Reply:
Sounds good to me. I had recently discovered that it is possible to include the CHANGELOG file directly as a part of the website too using a base64 encoding of its contents from the raw githubusercontent links (🤯). I will try doing that sometime soon